Libplanet/Net/Swarm.cs

```diff
@@ -224,6 +224,12 @@ static Swarm()
             private set;
         }
 
+        /// <summary>
+        /// The <see cref="BlockChain{T}"/> instance this <see cref="Swarm{T}"/> instance
```

Comment:
```suggestion
        /// The <see cref="BlockChain{T}"/> instance that synchronizes with
```

Reply:
Could you explain about your suggestion?

Reply:
I thought the suggested sentence is more natural, but please ignore it if you think there is no problem with the original sentence.